libraries/AP_Motors/AP_MotorsHeli_Quad.cpp

```diff
@@ -177,7 +177,10 @@ uint16_t AP_MotorsHeli_Quad::get_motor_mask()
     for (uint8_t i=0; i<AP_MOTORS_HELI_QUAD_NUM_MOTORS; i++) {
         mask |= 1U << (AP_MOTORS_MOT_1+i);
     }
```

Comment:
After this, it should do `mask = rc_map_mask(mask);` to get the correct outputs.

Reply:
I was wondering why that worked :)
"  map an internal motor mask to real motor mask... "
So does the "real" 32 bit mask returned by rc_map_mask() represent actual PWM output pins, and if so, where are they mapped to "real" channel numbers?

Reply:
The `rc_map_mask` is actually a `map_internal_motor_num_to_output_channel_mask`. Basically you can set one or more outputs to a motor, so this method uses the internal motor number to get the mask of outputs who's function is that motor. For example, you might set SERVO1_FUNCTION and SERVO9_FUNCTION to k_motor1, so the real mask needs to have this two outputs, not just output 1.